src/objects.js

```diff
@@ -1,41 +1,78 @@
 const createPerson = (name, age) => {
-  // your code here
+  const newPerson = new Object();
+  newPerson.name = name;
+  newPerson.age = age;
+
+  return newPerson;
```

Comment:
There is a cleaner way to do this, you could just return a new object without declaring it in a variable:
```
  return {
    name: name,
    age: age
  };
```
This can be shortened using the modern syntax (since the key and value are the same here) to this:
```
  return {
    name,
    age
  };
```